src/sections/GCP-Diagram/diagram.js

```diff
@@ -129,37 +131,11 @@ const DiagramWrapper = styled.div`
 
 const Gcp = () => {
 
-
```

Comment:
How is this being determined on other pages? Is this method consistent?

Reply:
Yes! It is , I have made sure the UI looks good in response to different theme.